### DISCUSSION TITLE
Can xDrip turn flashing lights or something similar on for an a alert?

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2637] [Q&A]
There is a request for this on facebook through Nightscout.
But, I thought xDrip can use tasker.  But, I myself have no experience with tasker.

Is there a way to activate something in the room the child is sleeping?
This is a child who does not wake up by the alert.  So, the father wants flashing lights.

**Comment by rudysroost** [comment]
Can we make xDrip work with IFTTT? I would be thrilled!

**Comment by jonprogers** [comment]
To achieve this I run a script to pull my BG level from Nightscout, into OpenHAB (home automation) so that 'extra' notification actions can be generated there...

**Reply by Navid200** [reply]
Yes, and I have already mentioned that is the request posted on facebook.

But, I am wondering if Nightscout is a necessity in accomplishing this considering we have tasker.

If you can help the individual who asked the question on facebook, it will be wonderful:
https://www.facebook.com/groups/cgminthecloud/posts/5855991071188767/

**Reply by jonprogers** [reply]
@Navid200 I'm sure nightscout isn't necessary, it's just how I did it - there are many ways to achieve most things!

I'll head over to Facebook when I can and see if I can add anything...

**Reply by Navid200** [reply]
I am wondering if there is an alternative way to Nightscout for accomplishing this considering we have the tasker feature in xDrip.

**Reply by jonprogers** [reply]
@Navid200 I'm so sorry if my post didn't sound constructive, I am apologetic that the way I do it doesn't use tasker as I expect that may be a better solution and more suited to most users, it's just not what I did. I too have no experience of tasker, that may be why I didn't even try to use it. I try to avoid facebook so am not as up to speed with conversations there, please accept my apologies for that.

**Reply by Navid200** [reply]
Sorry, hadn't had my coffee yet.  :o)  Thanks for your help

**Reply by jonprogers** [reply]
@Navid200 No worries, I can't type before I have mine so that saves me! Have a good day...
